spec/models/remote_translation_spec.rb

```diff
@@ -49,6 +49,14 @@
     expect(remote_translation).to be_valid
   end
 
+  it "checks app locales that need to be mapping" do
+    allow(RemoteTranslations::Microsoft::AvailableLocales).to receive(:remote_available_locales)
+                                                          .and_return(["pt"])
```

Comment:
This test passes even without the changes in this pull request :thinking:. I guess we need to either add `allow(RemoteTranslations::Microsoft::AvailableLocales).to receive(:available_locales).and_call_original` or change the way we handle tests with the `:remote_translations` tag so they stub `remove_available_locales` instead of stubbing `available_locales`.